Core/Src/monitor.c

```diff
@@ -139,6 +147,71 @@ void vPedalsMonitor(void* pv_params)
 		// MAX_ADC_VAL_12B) && 	!is_timer_active(&diff_timer)) 	start_timer(&diff_timer,
 		// PEDAL_FAULT_TIME); else 	cancel_timer(&diff_timer);
 
+
+		/* Evaluate accelerator faults */
+		if (is_timer_expired(&oc_timer))
+			//todo queue fault
+			continue;
```

Comment:
I don't think you want continue here, it will jump to the next loop of the entire task's for loop so it won't actually push that data to torque. Definitely a discussion of "do we want to actually push potentially bad data to torque" but if we low pass filter the torque and pedal values it should be fine, as long as the filter is longer than the debounce time for faults

Reply:
removed continue